src/api/aggregation/index.test.ts

```diff
@@ -1,13 +1,13 @@
 import { jest, describe, beforeEach, it, expect } from '@jest/globals';
 import { EventEmitter } from 'events';
 import {
-  subscribeToNewAttestations,
-  unsubscribeFromNewAttestations,
+  subscribeToNewAggregationReceipts,
+  unsubscribeFromNewAggregationReceipts,
 } from './index';
 import { ZkVerifyEvents } from '../../enums';
 import { ApiPromise } from '@polkadot/api';
 
-describe('subscribeToNewAttestations', () => {
+describe('subscribeToNewAggregationReceipts', () => {
```

Comment:
You'll probably need to do a domain registration in beforeAll

Reply:
Its mocked here, definitely needs a few more tests in future though